docs/getting-started/authenticating-to-github.md

```diff
@@ -16,4 +18,13 @@ Before you authenticate, you must already have a GitHub or GitHub Enterprise acc
 3. Click the **Connect** link in the GitHub section. If you are already connected to a GitHub instance and want to connect to another, this link will not be visible; instead click **Manage Connections** and then **Connect to GitHub**.
 ![Connect to GitHub in the manage connections dropdown in the Team Explorer pane](images/connect_to_github.png)
 4. In the **Connect to GitHub dialog** choose **GitHub** or **GitHub Enterprise**, depending on which product you're using.
-5. Type your credentials, then click **Sign In**.
+
+- **GitHub option**:  
+  ![Connect to GitHub dialog view](images/connect-to-github.png)
```

Comment:
This link is broken - the filename is `connect_to_github.png`.

Reply:
Oops. Put the image in the wrong directory.